dev/com.ibm.websphere.security/src/com/ibm/ws/security/authentication/utility/SubjectHelper.java

```diff
@@ -37,6 +37,12 @@
 public class SubjectHelper {
     private static final TraceComponent tc = Tr.register(SubjectHelper.class);
 
+    /**
```

Comment:
This comment seems to be extra and it appears that it was intended for the AuthenticationConstants.java class, which has this new comment already.